src/backend/DocDownloaderRedisCache/src/main/java/ca/bc/gov/ag/courts/controller/JobController.java

```diff
@@ -0,0 +1,84 @@
+package ca.bc.gov.ag.courts.controller;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.ag.courts.model.Job;
+import ca.bc.gov.ag.courts.repo.JobRepository;
+
+@RestController
+public class JobController {
+
+	Logger logger = LoggerFactory.getLogger(JobController.class);
+
+	@Autowired
+	JobRepository repo;
+
+	@GetMapping("/jobs")
```

Comment:
This question is probably related to all endpoints and how we return back the results. As I see from the code we return simple `List<Jobs` without any metadata. Do we want to get such metadata to analyze the response in a much granular manner? Like instead of simple `List<Jobs>` do we want to consider something like:

```json
{
  "data": {...},
  "status": "any kind of status for the data request, not for the connection itself",
  "error_message": "any comment for better debugging"
}
```

Reply:
Based on the JIRA task I will add "Add Springboot global exception handling for 40x and 50x errors" in small PRs. For Data exceptions I added IllegalArgumentException with a message, so the response is similar to other projects and since Redis operations are going to be a simple key-value CRUD.

Reply:
@eugenezimin - Have you studied the Job class. I have added many attributes which are just my first stab at what a job status should contain as far as metadata. Have a look if you haven't.

Reply:
Yup Shaun, thank you - I was reviewing it without having in mind that class you pointed out. 

Again, thanks!